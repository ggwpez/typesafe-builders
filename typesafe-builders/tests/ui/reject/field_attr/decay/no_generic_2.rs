mod other {
	use typesafe_builders::prelude::*;

	#[derive(Builder)]
	pub struct Struct {
		#[builder(decay)]
		x: &'static u8,
	}
}

fn main() {
}
