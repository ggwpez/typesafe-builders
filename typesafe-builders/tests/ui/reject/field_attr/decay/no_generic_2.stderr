error: Expected a path type
 --> tests/ui/reject/field_attr/decay/no_generic_2.rs:7:6
  |
7 |         x: &'static u8,
  |            ^^^^^^^^^^^
